domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/showcondition/ShowConditionParser.java

```diff
@@ -19,10 +19,12 @@
 
     public ShowCondition parseShowCondition(String rawShowConditionString) throws InvalidShowConditionException {
         try {
-            String[] andConditions = rawShowConditionString.split(AND_CONDITION_REGEX);
-            Optional<ShowCondition> optShowCondition = buildShowCondition(andConditions);
-            if (optShowCondition.isPresent()) {
-                return optShowCondition.get();
+            if (rawShowConditionString != null) {
```

Comment:
I did open a ticket to fix this:
https://tools.hmcts.net/jira/browse/RDM-2975
I'll link that ticket to this story then

Reply:
ok 👍